docs/preview/02-Guides/shared-access-key-authentication-with-azure-key-vault-integration.md

```diff
@@ -0,0 +1,159 @@
+# Shared access key authentication with Azure Key Vault integration
+The Arcus shared access key authentication is an API security filter to easily and safely share the same access key across a set of API endpoints. This kind of authentication is very common and therefore a popular feature in the Arcus WebApi library.
+
+The API security filter makes use of the [Arcus secret store](https://security.arcus-azure.net/features/secret-store) to retrieve the access keys. This makes sure that the secrets are safely accessed in the application.
+
+This user guide will walk through the process of adding shared access key authentication to an existing Web API application, using Azure Key Vault as location to store the access keys.
+
+## Terminology
+To fully understand this user guide, some terminology is required:
+* **Shared access key**: a single secret that's being used as the authentication mechanism of many API endpoints. Using a single secret means that there's also a single authorization level.
+* **Arcus secret store**: the secret store is a concept in the Arcus Security library to centralize secrets in an application ([more info](https://security.arcus-azure.net/features/secret-store)).
```

Comment:
I think this last item is a bit confusing.  Secret Store is a concept which 'abstracts' how secrets are stored, no ?  Then, I think the part 'to centralize secrets in an application', is not really clear.
I would change this to:
```suggestion
* **Arcus secret store**: the secret store is a concept in the Arcus Security library that represents an system that safely holds or stores secrets ([more info](https://security.arcus-azure.net/features/secret-store)).
```

Reply:
In the Arcus Security user guide about secret store integration, we used this: **Secret store: central place where the application retrieve its secrets.**, so I kinda used that.

Reply:
Changed in https://github.com/arcus-azure/arcus.webapi/pull/374/commits/7cfa8efcec6675f5657c4d7d2c33786b23db5996